go-demo/Makefile

```
@@ -0,0 +1,22 @@
+.PHONY: help
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+# Always (re)generate version.json via a phony "version" step so builds
+# get fresh metadata even if version.json already exists.
+.PHONY: version
+version.json: ## Generate/refresh version.json metadata
```

```suggestion
version: ## Generate/refresh version.json metadata
```

We want the target to match the `.PHONY: version`